Documentation/doc/Documentation/Developer_manual/Chapter_checks.txt

```diff
@@ -156,53 +156,8 @@ the computation is done with some exact number type.
 \section secchecks_controlling Controlling checks at a finer granularity
 
 The macros and related compile-time flags described so far all operate
-on the whole library. Sometimes the user may want to have a more
-selective control.  \cgal offers the possibility to turn checks on
-and off on a per-package basis. Therefore a package-specific term is
-inserted in the macro names directly after the \cgal prefix,
-<I>e.g.</I>, `CGAL_kernel_assertion(<Cond>)`. Similarly, the uppercase
-term is used for the compile-time flags;
-<I>e.g.</I>, `CGAL_KERNEL_NO_WARNINGS` switches off the warnings
-in <I>only</I> the kernel. Other packages have their own specific
-terms as documented in the corresponding chapters of the
-reference manual.
-
-For a new package you will first have to create a suitable header file
-with all macro definitions.  This is done with the shell script
-<TT>cgal_create_assertions.sh</TT>, to be found in the
-<TT>scripts</TT> directory.
-
-The following command will create a file <TT>optimisation_assertions.h</TT>:
-
-<CENTER>
-  <TT>sh cgal_create_assertions.sh optimisation</TT>
-
-</CENTER>
-
-You should place the generated file in the proper directory (and possibly
-rename it).  Then you can use the checks in the following fashion.
-
-\code{.cpp}
- #include <CGAL/Optimisation/assertions.h>
-
- void optimisation_foo( int i)
- {
-   CGAL_optimisation_precondition_msg( i == 42, "Only 42 allowed!");
-   // ...
- }
-\endcode
```

Comment:
This is the change that is responsible for the following change of dependencies, reported by the CI:
```
Differences in Optimisation_basic: Installation
Kernel_23
Profiling_tools
STL_Extension have disappeared.
```

In CGAL `master`, the header `<CGAL/Optimisation/assertions.h>` is documented in that page `Documentation/doc/Documentation/Developer_manual/Chapter_checks.txt`. See the [generated doc from CGAL-5.5](https://doc.cgal.org/5.5/Manual/devman_checks.html#secchecks_controlling).

In this PR, you have removed most of that section, and in particular that line documenting (in a way) the header `<CGAL/Optimisation/assertions.h>`. After the change this PR, the package `Optimisation_basic` no longer has any documented header, and its `dependencies` file is empty (or removed).